controller/src/main/java/com/flipkart/varadhi/controller/TopicLimitService.java

```diff
@@ -0,0 +1,10 @@
+package com.flipkart.varadhi.controller;
+
+public interface TopicLimitService {
```

Comment:
what is the usecase?
Is this giving us the quota allowed for topic?

Reply:
Yes, for getting the topic quota. Moved to `capacity` package under `core` module.